src/app/core/admin/admin-entity/admin-entity.component.ts

```diff
@@ -58,9 +61,10 @@ export class AdminEntityComponent implements OnInit {
   entityConstructor: EntityConstructor;
   private originalEntitySchemaFields: [string, EntitySchemaField][];
 
-  configDetailsView: DynamicComponentConfig<EntityDetailsConfig>;
+  configDetailsView: DynamicComponentConfig<any>;
```

Comment:
maybe we can keep this slightly more typed for now?

```suggestion
  configDetailsView: DynamicComponentConfig<EntityDetailsConfig | NoteDetailsConfig>;
```